include/zephyr/arch/arm/aarch32/cortex_a_r/scripts/linker.ld

```diff
@@ -142,6 +142,7 @@ SECTIONS
         *(.text)
         *(".text.*")
         *(.gnu.linkonce.t.*)
+        KEEP(*(.shared_irq))
```

Comment:
ditto. Add a guard on `CONFIG_ISR_SHARE_IRQ`. Same for all the other linker changes.

Reply:
Also changed.